src/css/profile/mobile/common/dropdownmenu.less

```diff
@@ -213,6 +214,8 @@
 		list-style: none;
 		padding: 0;
 		margin: 0;
+		max-height: calc(~"100vh - "2 * @dropdown-list-margin);
+		overflow-y: auto;
 		background-color: var(--dropdown-menu-options-background);
 		border-radius: 26 * @px_base;
 		box-shadow: inset 0 -3 * @px_base 0 0 @dropdown-list-box-shadow-color;
```

Comment:
My comment is out of scope this change but I think that the `box-shadow` is different as in guideline. `0px 0px 3px 0px rgba(0,0,0,35)`

Reply:
css box-shadow has different kind of two first parameters (not including inset option). They are (x,y) positions of shadow box not size of border in px. That's why we can not apply them one by one.
I assume the guide intention of
`Box-shadow: 0px 0px 3px 0px rgba(0,0,0,35)`
is to add shadow on each edge, probably starting clockwise.

Reply:
https://developer.mozilla.org/pl/docs/Web/CSS/box-shadow

Reply:
Ok will try this.

Reply:
I applied this change, only rounded borders are having shadow.. Added separate commit fixing this. Thank you.